src/patternfly/components/CustomSearchInput/examples/CustomSearchInput.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Custom search input'
+beta: true
+section: components
+cssPrefix: pf-c-custom-search-input
+---
+
+## Examples
+```hbs title=Basic
+{{#> custom-search-input custom-search-input--placeholder="Find by name"}}
+{{/custom-search-input}}
+```
+
+```hbs title=No-match
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="Joh"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-result-count
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="John Doe" custom-search-input--count="3"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-navigable-options
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="John Doe" custom-search-input--count="1/3" custom-search-input--IsNavigable="true" custom-search-input--IsFirstMatch="true"}}
+{{/custom-search-input}}
+```
+
+### Accessibility
+| Attributes | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-hidden="true"` | `.pf-c-custom-search-input__icon > *` | Hides the search icon from assistive technologies. **Required** |
+| `aria-label="Previous"` | `.pf-c-custom-search-input__nav > .pf-c-button` | Provides an accessible label for the previous nav button. **Required** |
+| `aria-label="Next"` | `.pf-c-custom-search-input__nav > .pf-c-button` | Provides an accessible label for the next nav button. **Required** |
+| `aria-label="[descriptive text]"` | `.pf-c-custom-search-input__text-input` | Provides an accessible label for the search input. **Required** |
+| `aria-label="Clear"` | `.pf-c-custom-search-input__close > .pf-c-button` | Provides an accessible label for the close button. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-custom-search-input` | `<div>` | Initiates the custom search input component. **Required** |
+| `.pf-c-custom-search-input__text` | `<span>` | Initiates the text area. **Required** |
+| `.pf-c-custom-search-input__text-input` | `<input>` | Initiates the search input. **Required** |
+| `.pf-c-custom-search-input__icon` | `<span>` | Initiates the search icon container. **Required** |
+| `.pf-c-custom-search-input__utilities` | `<span>` | Initiates the utilities area beside the search input. |
+| `.pf-c-custom-search-input__count` | `<span>` | Initiates the item count container. |
+| `.pf-c-custom-search-input__nav` | `<span>` | Initiates the navigable buttons container. |
+| `.pf-c-custom-search-input__close` | `<span>` | Initiates the close button container. **Required when there is text in the search input** |
```

Comment:
Is it closing the search input? or is it removing the text that is currently inside of the search input ?

Reply:
oops, this should have been renamed to `__clear`